ds-card-reader/driver/driver.go

```diff
@@ -34,6 +35,9 @@ func (drv *CardReaderDriver) Initialize(lc logger.LoggingClient, asyncCh chan<-
 	// Only setting if nil allows for unit testing with VirtualBoard enabled
 	if drv.Config == nil {
 		drv.svc = service.RunningService()
+		if drv.svc == nil {
+			return errors.New("custom card reader driver service is nil")
```

Comment:
I think we should be using fmt.Errorf rather than errors.New. @brian-intel, i think we a talk on this, where the errors package is being deprecated  by golang....

Reply:
We use both. `fmt.Errorf` if you are adding context via a format string or errors.New otherwise.